shared/src/commonMain/kotlin/edu/usf/cutr/otp/planner/api/PlannerApi.kt

```diff
@@ -0,0 +1,93 @@
+package edu.usf.cutr.otp.planner.api
+
+import edu.usf.cutr.otp.ApplicationDispatcher
+import edu.usf.cutr.otp.planner.model.Planner
+import edu.usf.cutr.otp.planner.model.RequestParameters
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import io.ktor.client.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import kotlinx.serialization.json.Json
+
+class PlannerApi(private val host: String,
```

Comment:
Could you rename this to `PlanApi(` to make it more obvious what is it, in terms of matching to the API named `/plan`?